pennylane/templates/embeddings/angle.py

```diff
@@ -43,6 +43,33 @@ class AngleEmbedding(Operation):
             with :math:`N\leq n`
         wires (Iterable): wires that the template acts on
         rotation (str): type of rotations used
+
+    Example:
+
+        Angle embedding encodes the features as rotaion angles of the specified rotation angle.
+
+        .. code-block:: python
+
+            import pennylane as qml
```

Comment:
We could potentially assume that the import was made by the user. This just makes the example shorter.
```suggestion
```